dubbo-registry/dubbo-registry-api/src/main/java/org/apache/dubbo/registry/client/metadata/store/InMemoryWritableMetadataService.java

```diff
@@ -201,25 +201,25 @@ public boolean unsubscribeURL(URL url) {
     }
 
     @Override
-    public void publishServiceDefinition(URL providerUrl) {
+    public void publishServiceDefinition(URL url) {
```

Comment:
it would be better to left this param named providerUrl which can prompt developer where this url from

Reply:
maybe not, this method can be invoke by consumer, it should not be defined provider.